GeoHealthCheck/enums.py

```diff
@@ -77,6 +77,9 @@
     'ESRI:FS': {
         'label': 'ESRI ArcGIS FeatureServer (FS)'
     },
+    'MapBox:TileJSON': {
+        'label': 'Mapbox TileJSON Service (TileJSON)'
+    },
```

Comment:
Official company name is `Mapbox` (not `MapBox`), make definition consistent: `Mapbox:TileJSON`.